app/jobs/generic_job.rb

```diff
@@ -134,11 +134,14 @@ def with_csv_items(csv, name:, druid_column: 'druid')
         cocina_object = Repository.find(druid)
         yield(cocina_object, csv_row, success, failure, row_num)
       rescue KeyError
-        raise "Column \"#{druid_column}\" not found"
+        raise
```

Comment:
Can we just do `failure.call(...)` here? I think that would be easier to follow.

Reply:
@justinlittman If this is for the whole sheet, can we validate it before making a background job (in the controller) and give instant feedback to the user?

Reply:
Yes, but ticketed separately: https://github.com/sul-dlss/dor-services-app/issues/3971

Even if that is done, this is still necessary as backup as there is nothing that guarantees that this will be invoked with a csv with a druid column.

Reply:
Can we just move the check to above the loop then?

Reply:
Yes. Done.